src/pudl/output/pudltabl.py

```diff
@@ -1138,34 +1138,21 @@ def plant_parts_eia(
         self,
         update: bool = False,
         update_gens_mega: bool = False,
-        update_true_gran: bool = False,
     ) -> pd.DataFrame:
         """Generate and return master plant-parts EIA.
 
         Args:
             update: If true, re-calculate the output dataframe, even
                 if a cached version exists.
             update_gens_mega: If True, update the gigantic Gens Mega table.
-            update_true_gran: If True, regenerate the True Granularities.
-
         """
-        # generate the true_gran table
-        # the true_gran table is really not helpful on it's own
-        if update_true_gran or self._dfs["true_grans_eia"] is None:
```

Comment:
Since the true_gran table isn't passed as an argument to `MakePlantParts.execute` anymore I took out this option to not regenerate the table. I could rework `TrueGranLabeler` and `MakePlantParts` a little so a separate table is passed in and then this argument could be implemented again. As it is, on the full plant parts list adding the true granularities takes about 2 minutes.